tests/algorithms/sbi/test_smcabc.py

```diff
@@ -0,0 +1,34 @@
+import pytest
```

Comment:
this replicates the testing code for `mcabc` almost 1-1. I would therefore vote for writing one test, e.g., `test_abc_run_scripts` or so, that makes the abc method an argument and parmaterizes it with pytest. E.g., 
```
from sbibm.algorithms.sbi.smcabc import run as run_smcabc
from sbibm.algorithms.sbi.mcabc import run as run_mcabc

...
@pytest.mark.parametrize("algorithm", (run_mcabc, run_smcabc))
...
def test_abc_run_scripts(task, algorithm, ...):
...
```

Reply:
fair enough, done with 08f8324